notebooks/additional-methods.ipynb

```diff
@@ -0,0 +1,589 @@
+    "\n",
+    "CAR specifically defines 100 year storage as \"completely mitigat[ing] an equal GHG emission\n",
+    "elsewhere.\" In other words, CAR claims that storing 1 tCO2 for 100 years (100 ton-years of storage)\n",
+    "is equivalent to a 1 tCO2 emission. Storing 1 tCO2 for 1 years (1 ton-years) would only provide\n",
+    "1/100th of the benefit needed for equivalence. Similarly, storing 1 tCO2 for 20 years (20 ton-years)\n",
+    "would provide 1/5th of the benefit needed for equivalence.\n",
+    "\n",
+    "The CAR ton year accounting method is in the Soil Enrichment Protocol (V1) in section 3.5.5:\n",
+    "https://www.climateactionreserve.org/wp-content/uploads/2020/10/Soil-Enrichment-Protocol-V1.0.pdf\n",
+    "\n",
+    "We show two examples of the `tonyear` module's calculations below for CAR's method with a 100 year\n",
+    "time horizon.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "58315bac",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Discount rate: 0%\n",
+      "Delay: 1 year(s)\n",
+      "Baseline atmospheric cost: 52.36 ton-years\n",
+      "Benefit from 1tCO2 with delay: 0.52 ton-years\n",
+      "Number needed: 100.0\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "time_horizon = 100\n",
+    "discount_rate = 0\n",
+    "delay = 1\n",
+    "baseline = tonyear.get_baseline_curve(\"joos_2013\")\n",
+    "\n",
+    "tonyear.print_benefit_report(\n",
+    "    calculate_tonyears(\"car\", baseline, time_horizon, delay, discount_rate)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "953676fb",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Discount rate: 0%\n",
+      "Delay: 30 year(s)\n",
+      "Baseline atmospheric cost: 52.36 ton-years\n",
+      "Benefit from 1tCO2 with delay: 15.71 ton-years\n",
+      "Number needed: 3.3\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "time_horizon = 100\n",
+    "discount_rate = 0\n",
+    "delay = 30\n",
+    "baseline = tonyear.get_baseline_curve(\"joos_2013\")\n",
+    "\n",
+    "tonyear.print_benefit_report(\n",
+    "    calculate_tonyears(\"car\", baseline, time_horizon, delay, discount_rate)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de80774a",
+   "metadata": {},
+   "source": [
+    "In essence, the CAR method awards benefit linearly in proportion to the time_horizon of choice.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "61f23691",
+   "metadata": {},
+   "source": [
+    "## Proposed Quebecois method\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "da8e5241",
+   "metadata": {},
+   "source": [
+    "The governement of Quebec (specifically the Ministère de l’Environnement et de la Lutte contre les\n",
```

Comment:
It would be nice to ask the person who emailed us about the Quebec method if we could acknowledge them. If they agree, we could mention their name here.

Reply:
That's a nice idea. Noting down to ask him in my reply.